piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -429,45 +439,58 @@ that r may know what his pleasure and counsel is concerning me, on whom I wait t
 for he is my life, and length of my days; and as I said before, I came at his command,
```

Comment:
remove the quotes (apostrophes) at the end of the first two paragraphs of this letter)